src/components/Button/Button.tsx

```diff
@@ -1,12 +1,79 @@
 interface ButtonProps {
   text: string;
   type: "anchor" | "button";
+  className?: string;
 }
+import { Link } from "react-router-dom";
+const register = "/register";
 
-export function Button({ text, type }: ButtonProps) {
+export function ButtonLogin({ text, type }: ButtonProps) {
+  const login = "/login";
+
+  if (type === "anchor") {
+    return (
+      <Link
+        to={login}
+        className="flex h-10 w-24 items-center justify-center rounded-lg bg-white font-bold text-blue-600"
+      >
+        {text}
+      </Link>
+    );
+  }
+
+  return (
+    <Link
+      to={login}
+      className="h-10 w-24 rounded-lg bg-white p-2 font-bold text-blue-600"
+    >
+      {text}
+    </Link>
+  );
+}
+
+export function ButtonRegister({ text, type }: ButtonProps) {
+  if (type === "anchor") {
+    return (
+      <Link
+        to={register}
+        className="flex h-10 w-24 items-center justify-center rounded-lg bg-white p-2 font-bold text-blue-600
+        "
+      >
+        {text}
+      </Link>
+    );
+  }
+
+  return (
+    <Link
+      to={register}
+      className="flex h-10 w-24 items-center justify-center rounded-lg bg-white p-2 font-bold text-blue-600
+      "
+    >
+      {text}
+    </Link>
+  );
+}
+
+export function ButtonCreate({ text, type }: ButtonProps) {
   if (type === "anchor") {
-    return <a href="#">{text}</a>;
+    return (
+      <Link
+        to={register}
+        className="`w-50 ${className}` rounded-lg bg-white p-2 text-xl font-bold text-blue-600
```

Comment:
The `className` variable doesn't exists and you're trying to use a template string ( `${}` ) inside a "normal" string, so the `w-50 ${className}` is appearing in the HTML: 

![image](https://github.com/hawks-atlanta/frontend-react/assets/62714297/587076af-6517-4752-b22c-80fee1f15682)